src/plugins/controls/public/actions/delete_control_action.test.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { BehaviorSubject } from 'rxjs';
+
+import { coreMock } from '@kbn/core/public/mocks';
+import { dataPluginMock } from '@kbn/data-plugin/public/mocks';
+import { dataViewPluginMocks } from '@kbn/data-views-plugin/public/mocks';
+import { ErrorEmbeddable } from '@kbn/embeddable-plugin/public';
+import { ViewMode } from '@kbn/presentation-publishing';
+
+import { getOptionsListControlFactory } from '../react_controls/controls/data_controls/options_list_control/get_options_list_control_factory';
+import { OptionsListControlApi } from '../react_controls/controls/data_controls/options_list_control/types';
+import {
+  getMockedBuildApi,
+  getMockedControlGroupApi,
+} from '../react_controls/controls/mocks/control_mocks';
+import { pluginServices } from '../services';
+import { DeleteControlAction } from './delete_control_action';
+
+const mockDataViews = dataViewPluginMocks.createStartContract();
+const mockCore = coreMock.createStart();
+
+const dashboardApi = {
+  viewMode: new BehaviorSubject<ViewMode>('view'),
+};
+const controlGroupApi = getMockedControlGroupApi(dashboardApi, {
+  removePanel: jest.fn(),
+  replacePanel: jest.fn(),
+  addNewPanel: jest.fn(),
+  children$: new BehaviorSubject({}),
+});
+
+let controlApi: OptionsListControlApi;
+beforeAll(async () => {
+  const controlFactory = getOptionsListControlFactory({
+    core: mockCore,
+    data: dataPluginMock.createStartContract(),
+    dataViews: mockDataViews,
+  });
+
+  const uuid = 'testControl';
+  const control = await controlFactory.buildControl(
+    {
+      dataViewId: 'test-data-view',
+      title: 'test',
+      fieldName: 'test-field',
+      width: 'medium',
+      grow: false,
+    },
+    getMockedBuildApi(uuid, controlFactory, controlGroupApi),
+    uuid,
+    controlGroupApi
+  );
+
+  controlApi = control.api;
+});
+
+test('Action is incompatible with Error Embeddables', async () => {
```

Comment:
Is this test case still needed? Control can never be an error embeddable.

Reply:
Good catch - removed in 3917c17deee81a3071efbb8d5e40fb540583f1d4 :+1: